app/packs/stylesheets/work.scss

```diff
@@ -47,6 +47,10 @@
       margin-bottom: 0rem;
     }
 
+    th {
```

Comment:
We don't want to apply this to the files table and only want to apply it to the first column of each table.  How about adding the `col-3` class just to the `th` tags that need it?

Reply:
Ok, switched to using col-3 class where needed instead of in css